spark_parser/spark.py

```diff
@@ -494,11 +500,16 @@ def makeSet(self, token, sets, i):
 
             if parent == i:
                 continue
-
+            if self.debug['reduce']: #STRANIKS_SCAN 01/03/2016
```

Comment:
I've been worked in offline without the www, in such cases comment very needed, I'm sorry, I had to clean the code before downloading =)))